### ISSUE TITLE
How to test the results of pix4pont on S3DIS (Area 5) Segmentation？

### ISSUE THREAD

**Issue by Daybreak-Zheng** [guochengqian/PointNeXt#115]
**May I ask how to test the results of pix4pont on S3DIS (Area 5) Segmentation？**

![image](https://user-images.githubusercontent.com/44741080/225802451-aeafac5b-45de-4ebd-9a17-2aedeabf8cf8.png)

How to get the results of pix4pont？

An error was encountered while testing with this command:

`CUDA_VISIBLE_DEVICES=0 bash script/main_segmentation.sh cfgs/s3dis_pix4point/pix4point.yaml wandb.use_wandb=False mode=test --pretrained_path pretrained/s3dis_pix4point/s3dis_pix4point-finetune_encoder-pix4point-ngpus2-seed697-20230316-012711-DF3tvptwejq9cxoQPMUcPp/checkpoint/s3dis_pix4point-finetune_encoder-pix4point-ngpus2-seed697-20230316-012711-DF3tvptwejq9cxoQPMUcPp_ckpt_best.pth`

There seems to be no test set in cfgs/s3dis_pix4point/default.yaml, and no test_area: 5 is specified as in cfgs/s3dis/default.yaml

![image](https://user-images.githubusercontent.com/44741080/225804512-5b280506-c759-4bf0-aa75-65c53e99f926.png)

![image](https://user-images.githubusercontent.com/44741080/225804459-1177f5bf-1c83-4387-a34c-06e78117d6c7.png)

**How do I test after training pix4point, and what are the test commands?**

**How to test the results of pix4pont on S3DIS (Area 5) Segmentation?**

Looking forward to receiving your reply!

**Comment by guochengqian** [comment]
Closing as answered/stale. Pix4Point S3DIS should follow the S3DIS segmentation test flow with the Pix4Point config/checkpoint. The v0.1.1 FAQ now covers the general S3DIS test/visualization paths.
